newscorp-ghfb/newskit
# feat(PPDSC-2218): added Scroll storybook scenario

PPDSC-2218

**What**

1. Background - why this is needed
Missing scenario in the scroll component storybook showcasing how users can scroll inside the container using tab key.

2. What did you do
Added a scenario to the scroll component storybook that allow the user to scroll inside of the container using the tab key.

3. What does the reviewers should expect
A scenario where focus moves to first interactive element inside the container and then through the other interactive elements.





**I have done:**

**I have tested manually:**
- [x] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.




**After:**
![Screenshot 2022-07-05 at 09 17 49](https://user-images.githubusercontent.com/92727370/177282991-32fd0b28-d063-4748-ac6e-3a400da3edb5.png)





**Who should review this PR:**


**How to test:**
Please check scenario 'scroll-inside-the-container-with-tab-key' in the scroll component storybook
